### ISSUE TITLE
Why choose JDC to extract pitch?

### ISSUE THREAD

**Issue by Cheneng** [yl4579/StyleTTS2#129]
I am curious about the choice of pitch extractor, because in the sovits project, the accuracy of the pitch almost directly affects the accuracy of the pronunciation. I would like to ask why you choose JDC, cause it is good enough?

**Comment by yl4579** [comment]
It was just a model I found to work quite well for pitch extraction since the StarGANv2-VC project: https://github.com/yl4579/StarGANv2-VC
You can definitely replace it with other stuff, but since the model is trained end-to-end with the pitch extractor (unlike sovit where the pitch is extracted first as features), the performance of pre-trained model doesn't matter that much as it will be fine-tuned with the dataset you are using.
